Counties/Florida/Bay County/Scraper/Scraper.py

```diff
@@ -301,7 +310,10 @@ def search_portal(case_number):
     Performs a search of the portal from its home page, including selecting the case number input, solving the captcha
     and pressing Search. Also handles the captcha being solved incorrectly
     :param case_number: Case to search
-    :return: True if a valid case was found, False if not.
+    :return: A dictionary with 3 keys: 'found', 'count', 'cases'.
```

Comment:
Would a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) be a better fit for this?

Reply:
Actually, couldn't you just return `cases`? You could then derive `found` by checking the truthiness of `cases` with `if cases` or whatever, and you could get `count` via `len(cases)`.